granulate_utils/linux/process.py

```diff
@@ -128,3 +128,7 @@ def is_process_basename_matching(process: psutil.Process, basename_pattern: str)
         return True
 
     return False
+
+
+def is_kernel_thread(process: psutil.Process) -> bool:
```

Comment:
This is the logic from `pgrep_exe` right?

Reply:
Make sure to update `pgrep_exe` to use it

Reply:
> This is the logic from `pgrep_exe` right?

Yes.

> Make sure to update `pgrep_exe` to use it

Already did:
https://github.com/Granulate/gprofiler/pull/554